scripts/glutton/update/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "glutton-update-bin"
+version = "1.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "Script to update Glutton parachains storage values"
+
+
+[[bin]]
+name = "glutton-update"
+path = "src/main.rs"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false }
+subxt = { version = "0.29.0", features = ["unstable-metadata"] }
+sp-keyring = "23.0.0"
```

Comment:
I think we can nix this dep, don't think it's needed. Seems to compile on my machine without it.
```suggestion
```